home.html

```diff
@@ -10,16 +10,13 @@
 <body>
     <header>
         <nav>
-            <ul>
-                <li><a href="home.html" class="home">Home</a></li>
-                <li><a href="products.html" class="shop">Shop</a></li>
-                <li><a href="about-us.html" class="about">About Us</a></li>
-                <li><a href="contact.html" class="contact">Contact</a></li>
-            </ul>
-
-            <div class="loginbar">
-                <li><a href="login.html" class="login">Login</a></li>
-                <li><a href="cart.html" class="cart">Cart</a></li>
+            <div class="nav-flexbox">
```

Comment:
Same thing here for the structure and flexbox use - refer to first comment about nav

Reply:
updated and sent push to repo